### DISCUSSION TITLE
v2.6.1

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#3971] [分享 / Share]
✨新增
- 优化 `简化评论区` 的时间样式. (#3541, PR #3791 by [Tinhone](https://github.com/Tinhone))
- `下载视频`:
  - 支持选择备用下载地址. (PR #3798 by [Seaward233](https://github.com/Seaward233))
  - 新增 `StreamSaver` 输出方式.
  - 新增 `Motrix` 输出方式. (PR #3908 by [FoundTheWOUT](https://github.com/FoundTheWOUT))
- `删除广告` 支持屏蔽动态的商品推荐. (#3778)
- 优化 `自定义顶栏` 的弹窗逻辑, *有任意元素聚焦时* -> *仅在输入框聚焦时*显示弹窗. (#3756)
- `展开弹幕列表` 新增 `有选集时不自动展开` 选项. (#1734)
- 新增功能 `禁止滚轮调音量`. (#3489)
> 在网页全屏 / 全屏模式下, 禁止鼠标滚轮控制播放器的音量.

- 新增功能 `批量导入合集`. (PR #3884 by [swhoro](https://github.com/swhoro))
> 在合集页面制作一个批量导入按钮，可以新增所有合集内视频至同名收藏夹

- `扩展倍速` 添加 `隐藏移除图标` 和 `隐藏新增图标` 选项. (PR #3864 by [Tinhone](https://github.com/Tinhone))
- 新增插件 `快捷键扩展 - 开关 CC 字幕`, 默认为 <kbd>Shift</kbd> + <kbd>C</kbd>. (#3407)
- 新增插件 `快捷键扩展 - 开关弹幕列表`, 默认为 <kbd>Shift</kbd> + <kbd>D</kbd>. (#3613)
- `新版本提示` 支持在安装前查看更新说明.
- 功能面板为空时增加引导提示. (#3932, PR #3936 by [Ethkuil](https://github.com/Ethkuil))
- `下载字幕` 支持在番剧页面使用.
- 新增功能 `禁用评论区搜索词`. (#3648)
> 禁用评论区的搜索词链接.

🐛修复
- `下载视频` 的 `flv` 格式恢复 1080P 支持. (一共就 360P / 1080P, 其他清晰度还是没有, 需要的话请使用 dash 格式)
- 修复评论区功能在新版播放页上的支持, 包括 `复制评论链接` / `简化评论区` 等. (#3807)
- 修复 `清爽首页` 的排行榜第三项数据不正确. (#3911, PR #3912 by [Colerar](https://github.com/Colerar))
- 修复 `简化直播间` 对左下角红包的简化样式. (#3243)
- MDI 图标库内联至本体中, 避免阻塞页面加载. (#3964, #3888)
- 修复 `侧栏垂直偏移` 在部分直播间失效的问题. (#3934)
- 修复 `自定义顶栏` 的收藏夹在加载完毕后仍显示加载中. (#3940)
- 修复 `展开弹幕列表` 检测到的弹幕量不正确. (获取页面的已装载弹幕量太坑了, 改为调 API 获取总弹幕量来判断了) (#3958)
- 修复 `启用弹幕空降` 失效. (#3187)
- 修复 `高分辨率图片` 在个人空间投稿页失效. (#3894)
- 修复 `复制评论链接` 在个人空间复制出的链接不正确. (#3821)
- 修复 ASS 弹幕中的颜色标记不正确. (#3960)

☕开发者相关
- 完成 define API + SwitchOptions API 的改造 (受影响组件为 `简化首页` 和 `简化直播间`), 感谢 [timongh](https://github.com/timongh) 的长期贡献.
- 重新拆分了一下 PlayerAgent 的文件.
- 修复 CI 构建脚本未提交新增文件. (#3903)
- 新增 Dialog API, 可用简单封装以弹窗形式展示的内容.
- 开源许可有所变更, 对再分发有一些限制, 详见 [LICENSE](./LICENCE.md).
- 删除了 `<template>` 元素多余的 class. (#3951)
- 修复 VSCode Task 中的 ESLint 命令和 package.json 中不一致. (#3950)
- TypeScript 升级至 4.9 版本.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.6.1'>v2.6.1</a>.</em>

**Comment by Geeyun-JY3** [comment]
YouTube 预设里没把「开关 CC 字幕」改成 <kbd>C</kbd> 吧：<https://github.com/the1812/Bilibili-Evolved/discussions/3407#discussioncomment-2979156>（别忘了禁用该预设里的「投币」，原先是继承默认的 <kbd>C</kbd>）

***

另外 PotPlayer 里显示/隐藏字幕的默认快捷键是 <kbd>Alt</kbd>+<kbd>H</kbd>，这还要加到 PotPlayer 的预设里吗？

***

[HTML5视频播放器增强脚本](https://github.com/xxxily/h5player) 就应该是没有开关字幕的快捷键：https://github.com/xxxily/h5player/issues/352

***

PotPlayer 和 HTML5Player 这两个预设在安装 `快捷键扩展 - 开关弹幕列表` 都应禁用该快捷键吧，毕竟 <kbd>Shift</kbd>+<kbd>D</kbd> 已经被用于弹幕开关了。

***

顺带发现 PotPlayer 和 HTML5Player 这两个预设的切换倍速对应的 <kbd>Z</kbd> 好像丢了。

**Reply by the1812** [reply]
@LonelySteve 切换倍速那个之前有吗, 我看目前代码里没写
![image](https://user-images.githubusercontent.com/26504152/222939780-fca88a65-a13c-4e6b-8e51-02852f457d92.png)

**Reply by LonelySteve** [reply]
> @LonelySteve 切换倍速那个之前有吗, 我看目前代码里没写 ![image](https://user-images.githubusercontent.com/26504152/222939780-fca88a65-a13c-4e6b-8e51-02852f457d92.png)

原先就没有

**Reply by Geeyun-JY3** [reply]
额我说的丢了就是说之前是不是忘了将 PotPlayer 和 HTML5Player 这两个预设的切换倍速设为 <kbd>Z</kbd>。
- PotPlayer：
  ![image](https://user-images.githubusercontent.com/49396421/228746643-1e33d3a1-ff94-4f7e-98ef-a905a9485fba.png)
- HTML5Player：https://h5player.anzz.top/home/Introduction.html#快捷键列表
  ![image](https://user-images.githubusercontent.com/49396421/228746486-c56b18c0-6948-445b-8203-d0235e71f2e5.png)
